src/crossfilter.js

```diff
@@ -121,6 +138,11 @@ function crossfilter() {
     offset = tmp.offset;
     one = tmp.one;
     zero = ~one;
+      
+    // Expose the dimension mask for internal use
+    dimension._offset = offset;
```

Comment:
Well, since map keys are coerced to strings we would need a unique string key for each dimension then.  And since the offset/one/zero is defined in the dimension constructor itself it would also mean that map in the crossfilter scope would need to be exposed.  So, it seemed cleaner to expose private offset/zero/one in the dimension then expose the private offset/zero/one map in the crossfilter scope...